docs/herramientas_test.md

```diff
@@ -0,0 +1,49 @@
+# Herramientas de test
+
+## Criterios de decisión
+
+Para elegir las diferentes herramientas para nuestro proyecto tendremos que tener en cuenta una seria de criterios que seran los siguientes:
+
+- **Compatibilidad**: La herramienta debe ser compatible con el lenguaje de programación que estamos utilizando en nuestro caso TypeScript.
+
+- **Valoración**: Se tendrá en cuenta la valoración que tiene la herramienta en [Snyk Advisor](https://snyk.io/advisor)
+
+- **Híbrido**: Se tendrá en cuenta que la herramienta tenga biblioteca para aserciones así como el test runner.
+
+
+## Herramientas de test
+
+Necesitamos elegir herramientas de test para nuestro proyecto. Para ello será necesario elegir un una biblioteca de aserciones, un test runner y un CLI para ejecutar los test.
+
+### Aserciones
+
+- [Chai](https://www.chaijs.com/): Chai es una biblioteca de aserciones para Node.js. Tiene una valoración de 97.
+
+- [Assert](https://nodejs.org/api/assert.html): assert es una biblioteca de aserciones para Node.js, esta es la biblioteca estandar de Node. Tiene una valoración de 86.
+
+Hay otras bibliotecas de aserciones, sin embargo como tienen una valoración baja no se tendrán en cuenta, como por ejemplo: 
+- [assertive](https://github.com/groupon/assertive): con una valoración de 49.
```

Comment:
Antes quitaste should.js que tenía más puntuación que esta, que no llega ni al aprobado. ¿Por qué la metes ahora?